.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: node_js
 
 node_js:
+  - iojs
+  - '0.12'
```

Comment:
I get the change in the harness, but I don't get the change in the `.travis.yml` file - why is this necessary?